app/jobs/user_subscribe_job.rb

```diff
@@ -0,0 +1,9 @@
+class UserSubscribeJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(name, email)
+    klass = Rails.env.test? ? FakeMailChimpClient : MailChimpClient
+    @client = klass.new
```

Comment:
If mailing is broken now I'd say go the concern route just to get practice with them and we can revisit.